src/styles/components/_component-grid.scss

```diff
@@ -1,13 +1,26 @@
 $breakpoint-tablet: 820px;
 
-.components-grid {
+.grid-container {
   width: 100%;
-  text-align: center;
-  padding: 1rem;
 
   @include background-pattern;
+}
+
+.components-grid {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: auto;
+  padding: 1rem;
+  padding-top: 4rem;
+  gap: 1rem;
+  width: 100%;
+  max-width: 100rem;
```

Comment:
100rem seems huge! Is that what we want? 75rem seems closer to what's in Figma.

Reply:
Rebased recently to include Marissa's homepage changes, and her max-width was 100 rem for the title card and this component grid. I do agree that 75rem's closer to figma, which has a width of 1440px with 120px margins, but it feels a bit out of scope to go tweaking those max-widths now that the homepage layout has been addressed in its own branch. 

What do you think? Should I go ahead and tweak these anyways?